code-for-charlottesville/Fire-Risk
# Merge data into big csv

## What is added?

- A script to download, merge, and index the data we will need for running machine learning and performing other relevant tasks for the cville fire department
- readme instructions on how to run the script

## How to test this PR

- checkout this branch
- follow instructions in the readme, look at the logs and be sure that no errors are logged
- change the postgres endpoint to `"postgres://postgres:putoutfires@firecodeforcville.curkiseapqty.us-east-1.rds.amazonaws.com:5432/firerisk"` and make sure that the script is successful. This is a deployed postgres instance, we can use this as a dev environment for working on this project.
